cslabs-backend/Controllers/UserModulesController.cs

```diff
@@ -0,0 +1,29 @@
+using System.Linq;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using CSLabsBackend.Models;
+
+namespace CSLabsBackend.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserModulesController : BaseController
+    {
+        public UserModulesController(DefaultContext defaultContext, IMapper mapper) : base(defaultContext, mapper)
+        {
+        }
+        // GET api/values
+        [HttpGet]
+        public IActionResult Get()
+        {
+            return Ok(DatabaseContext.UserModules.ToList());
```

Comment:
The user should not be able to see other user's module instances. You need to filter your query down to only user_modules that match the current user's id. Call `GetUser()` to get the current user's model. Use the current user's model to filter the user modules by the user's id. This will only work when the `[Authorize]` is added to the controller.